### ISSUE TITLE
missing oauth/token endpoints

### ISSUE THREAD

**Issue by fredriklindell** [benweier/blizzard.js#54]
newly implemented feature, credentials, in version 2.0.0 does not work for origin: eu, kr, tw.

Below endpoints does not exist:
https://eu.api.battle.net/oauth/token
https://kr.api.battle.net/oauth/token
https://tw.api.battle.net/oauth/token

Am I using wrong origins?

Thanks in advance!

**Comment by benweier** [comment]
Well shit. I'm missing some code from another machine. I'll release a patch fix pronto.

**Comment by fredriklindell** [comment]
No rush :) Nice work!

**Comment by benweier** [comment]
I'm so sorry I pushed a broken release @fredriklindell 

I've just published `2.0.1` to resolve this issue.

The OAuth token API is available for `us` (for US and SEA), `eu` (just Europe), and `apac` (for TW and KR). You would request application tokens for those geographic regions like this:

```
blizzard.credentials({ origin: 'us', id: BLIZZARD_CLIENT_ID, secret: BLIZZARD_SECRET_KEY })
  .then(response => console.log(response.data))
  .catch(err => console.log(err.response.data))
```
